src/NServiceBus.AmazonSQS.AcceptanceTests/Infrastructure/TestIndependenceMutator.cs

```diff
@@ -0,0 +1,22 @@
+﻿namespace NServiceBus.AcceptanceTests.Infrastructure
+{
+    using System.Threading.Tasks;
+    using NServiceBus.AcceptanceTesting;
+    using NServiceBus.MessageMutator;
+
+    class TestIndependenceMutator : IMutateOutgoingTransportMessages
```

Comment:
@SeanFeldman can you remind me why we are doing this as a mutator again and not the outgoing physical part in a behavior?